### ISSUE TITLE
CELA P1_A11y_code.visualstudio.com_WCAG2.1_Homepage_Zoom: When the page is zoomed to 400% then some of the Hyperlink present at the top disappears and is not present anywhere on the page

### ISSUE THREAD

**Issue by khansahab15** [microsoft/vscode-docs#3431]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-shbagl%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847575447&sdata=XQqvaO2rFh7p/7F2DmzFj2wgJSiYHGuJV73Xg8hPVbA%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11ySev1; #A11yTCS; #A11yWCAG2.1; #Benchmark; #CELA P1; #ChromiumEdge; #Zoom; #DesktopWeb; #MAS1.4.10; #Win10; #Visual Studio Code; #CELA P1_A11y_code.visualstudio.com_WCAG2.1

**Environment Details:**
Application Name: marketplace.visualstudio.com
#URL: https://code.visualstudio.com/docs
Windows Version: Windows10
Browser version: Microsoft Edge 80.0.361.5

**Repro Steps:** 
1.	Launch the application URL: https://code.visualstudio.com/docs
2.	Home screen will open
3.	Zoom the page to 400%.

**Actual:**
When the page is zoomed to 400% the some of the Hyperlink present at the top disappears and is not present anywhere on the page

**Expected:**
When the page is zoomed to 400% then some of the Hyperlink present at the top shouldn’t disappear

**User Impact:**
Loss of content will be there for Low vision users.

**Attachment:**
[#3431.zip](https://github.com/microsoft/vscode-docs/files/4265372/3431.zip)

**Comment by gregvanl** [comment]
Hi @khansahab15 I can't repro your issue. I see all the top navigation links from the right side "hamburger" dropdown. You will need to scroll down to see them all. Which items are missing?

**Comment by khansahab15** [comment]
Hi @gregvanl 
The issue still repro. To clarify when we zoom to 400%  **API, Extension, FAQ** items are missing.
The same is justified in the below attachment.

[3431_repro.zip](https://github.com/microsoft/vscode-docs/files/4305217/3431_repro.zip)

**Comment by gregvanl** [comment]
Hi @khansahab15 I can't repro this.

![image](https://user-images.githubusercontent.com/12818376/76991296-92b0c500-6906-11ea-826f-185512a335b8.png)

**Comment by gregvanl** [comment]
Closing at not able to repro

**Comment by kapilvaishna** [comment]
**GithubTags**:#Closed;
Issue not repro in environment https://code.visualstudio.com/docs, so closing the bug 

https://user-images.githubusercontent.com/124254981/216603697-07f58da7-4c48-4ffd-9816-3c1404366738.mp4

**Comment by 12-shweta** [comment]
#CELA_code.visualstudio.com_WCAG2.1_FEB2020;
